packages/calypso-products/src/plans-list.tsx

```diff
@@ -2262,6 +2279,16 @@ export const PLANS_LIST: Record< string, Plan | JetpackPlan | WPComPlan > = {
 		getPathSlug: () => 'wooexpress-small-yearly',
 	},
 
+	// Not a real plan. This is used to show the Plus offering in
+	// the main plans grid
```

Comment:
Correction: this is not the main plans grid, it's the Woo Express plans grid. :grin: